src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -115,11 +115,11 @@ public static function getStyleDefinition($name)
             self::$styles = self::initStyles();
         }
 
-        if (!self::$styles[$name]) {
-            throw new InvalidArgumentException(sprintf('Style "%s" is not defined.', $name));
+        if (isset(self::$styles[$name])) {
+            return self::$styles[$name];
```

Comment:
```php
public static function getStyleDefinition($name) {
...
   return $this->resolveStyle($name);
...
}
```
In this PR - i add covers and refactoring
@nicolas-grekas, You proposal convert resolveStyle to `static` and also change logic of `getStyleDefinition`

```php
     /**
      * @param string|TableStyle $name
      */
     public static function getStyleDefinition($name)
```

from
```php
     /**
      * @param string $name
      */
     public static function getStyleDefinition($name)
```
?